### PR TITLE
🌎 Clean up countries code used by various channel types

### PR DESCRIPTION
**Pull request by rowanseymour**
Fixes some odd country ordering too

Captura de Pantalla 2021-04-14 a la(s) 14 30 09

becomes

Captura de Pantalla 2021-04-14 a la(s) 14 32 15

### CODE REVIEW

File: temba/channels/tests.py [hunk 1]
```diff
@@ -865,10 +865,6 @@ def test_claim(self):
         self.login(self.admin)
         response = self.client.get(reverse("channels.channel_claim"))
         self.assertEqual(200, response.status_code)
-        self.assertEqual(
```
Comment (rowanseymour) [inline temba/channels/tests.py:868]: not used

File: temba/channels/types/plivo/views.py [hunk 2]
```diff
@@ -53,10 +75,10 @@ def pre_process(self, *args, **kwargs):
             return HttpResponseRedirect(reverse("orgs.org_plivo_connect"))
 
     def is_valid_country(self, country_code):
-        return country_code in PLIVO_SUPPORTED_COUNTRY_CODES
+        return country_code in CALLING_CODES
 
     def is_messaging_country(self, country):
-        return country in [c[0] for c in PLIVO_SUPPORTED_COUNTRIES]
```
Comment (rowanseymour) [inline temba/channels/types/plivo/views.py:59]: it's sorta funny how we convert this thing into a tuple with country names but then convert it back here on the fly..

File: temba/channels/views.py [hunk 3]
```diff
@@ -2005,16 +1413,23 @@ def get_address(self, obj):
...
-            country = forms.ChoiceField(choices=TWILIO_SEARCH_COUNTRIES)
+            country = forms.ChoiceField()
+
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
 
-        form_class = SearchNumbersForm
+                from temba.channels.types.twilio.views import SEARCH_COUNTRY_CHOICES
+
+                self.fields["country"].choices = SEARCH_COUNTRY_CHOICES
```
Comment (rowanseymour) [inline temba/channels/views.py:1430]: this is a temporary workaround until I move all these search views into their respective channel type packages

File: temba/utils/__init__.py [hunk 4]
```diff
@@ -161,15 +158,6 @@ def print_max_mem_usage(msg=None):
     print("=" * 80)
 
 
-def get_country_code_by_name(name):
```
Comment (rowanseymour) [inline temba/utils/__init__.py:164]: not used

File: temba/utils/countries.py [hunk 5]
```diff
@@ -0,0 +1,285 @@
+from django_countries.data import COUNTRIES
+
+from django.utils.translation import ugettext_lazy as _
+
+NAMES = COUNTRIES.copy()
+NAMES["GB"] = _("United Kingdom")
+NAMES["US"] = _("United States")
+NAMES["AC"] = _("Ascension Island")
```
Comment (rowanseymour) [inline temba/utils/countries.py:8]: fixed typo - was `Scension`

File: temba/channels/types/plivo/views.py [hunk 6]
```diff
@@ -52,11 +74,11 @@ def pre_process(self, *args, **kwargs):
...
             return HttpResponseRedirect(reverse("orgs.org_plivo_connect"))
 
-    def is_valid_country(self, country_code):
-        return country_code in PLIVO_SUPPORTED_COUNTRY_CODES
+    def is_valid_country(self, calling_code: int) -> bool:
+        return calling_code in CALLING_CODES
 
-    def is_messaging_country(self, country):
-        return country in [c[0] for c in PLIVO_SUPPORTED_COUNTRIES]
+    def is_messaging_country(self, country_code: str) -> bool:
```
Comment (rowanseymour) [inline temba/channels/types/plivo/views.py:80]: renamed args here to clarify that the first is a numerical calling code and the second is an iso country code

**Review by norkans7 — APPROVED** [review]

**Pull request merged**